lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -12,6 +13,8 @@
 
 from courseware.access import has_access
 
+UNNAMED_MODULE_NAME = _("(Unnamed)")
```

Comment:
I think it's only necessary when there is a potential question - this one seemed pretty straightforward.